src/app/page.tsx

```diff
@@ -12,6 +27,8 @@ export default function Home() {
           <Button className="bg-blue-700 text-slate-300">File</Button>
         </div>
         <div className="flex">
+          <Button onClick={signIn} className="bg-blue-700 text-slate-300">Sign In</Button>
```

Comment:
Add padding between the buttons so they don't appear squished
<img width="407" alt="image" src="https://github.com/CS-3311-JDE-3346/3346-vocoder-webapp/assets/47485510/de54ad40-ba8a-49ab-bdcc-83263f9cddc7">